ultralytics/data/utils.py

```
@@ -567,7 +567,7 @@ def check_cls_dataset(dataset: Union[str, Path], split: str = "") -> Dict:
             else:
                 LOGGER.info(f"{prefix} found {nf} images in {nd} classes ✅ ")
 
-    return {"train": train_set, "val": val_set, "test": test_set, "nc": nc, "names": names, "channels": 3}
+    return {"train": train_set, "val": val_set, "test": test_set, "nc": nc, "names": names, "channels": 1}
```

Pylint (3.3.7)

[convention] Line too long (106/100)

(C0301)